imagersite/imager_images/templates/photo.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block content %}
+
+<div class='one_photo'>
+
+<h2>{{ user.get_full_name }}</h2>
+
+<h2>{{ photo.title }}</h2>
+<div style="position:relative;">
+
+
+    <img src='/media/{{ photo.file }}'/>
```

Comment:
use `{{MEDIA_URL}}` please, or better yet `{{ photo.file.url }}` which should include the MEDIA_URL on its own.